plugin/src/main/java/com/denizenscript/denizen/objects/PlayerTag.java

```diff
@@ -1624,7 +1632,7 @@ else if (obj instanceof EntityTag) {
         // @attribute <PlayerTag.enderchest>
         // @returns InventoryTag
         // @description
-        // Gets the player's enderchest inventory.
+        // Returns the InventoryTag of the player's enderchest inventory.

```

Comment:
This is somewhere between pointless and actively making meta worse.

Reply:
I just made it match ya know https://puu.sh/GfVKg.png